### DISCUSSION TITLE
Make the jeb sheep drop a random coloured wool when sheared.

### DISCUSSION THREAD

**Discussion by glenngenre** [PurpurMC/Purpur#808] [__Completed__] [answered]
It doesn't have to be the exact same colour as the one that's being displayed when it's sheared, it just has to be random.

**Comment by BillyGalbreath** [comment, marked as answer]
Implemented in build 1470

https://github.com/PurpurMC/Purpur/commit/996ea58dbeec1cdeb94946e3ce6e7b226fad45db
